db/builder.cc

```diff
@@ -330,8 +335,6 @@ Status BuildTable(
         ignored = fs->DeleteFile(blob_file_path, IOOptions(), dbg);
         ignored.PermitUncheckedError();
       }
-
-      blob_file_additions->clear();
```

Comment:
It's possible that no sst is generated, but blob files are written and recorded in the MANIFEST. How will these blobs be accessed?

Reply:
Or is this just for the accuracy of stats, e.g. blobs written, blob_bytes, etc?

Reply:
> Or is this just for the accuracy of stats, e.g. blobs written, blob_bytes, etc?

Exactly, please see above.